src/rpc/hook.rs

```diff
@@ -27,4 +59,12 @@ impl HookService for HookRpc {
         })?;
         Ok(Response::new(hook::v0::EvalResponse { json }))
     }
+
+    async fn exit_process(
+        &self,
+        request: Request<hook::v0::ExitProcessRequest>,
+    ) -> Result<Response<hook::v0::ExitProcessResponse>, Status> {
+        self.notification("exitProcess", request).await?;
+        Ok(Response::new(hook::v0::ExitProcessResponse {}))
```

Comment:
In case this is not WIP, there is no Lua counterpart for `exitProcess` in the PR yet

Reply:
No need to review these yet. I haven't even tested that they work (which would have uncovered this 😁 )

Testing it a bit of a PITA since you have to fire up DCS etc so I prefer to write a bunch of these simple APIs then test them all together, fix any issues, push then move the PR out of draft mode.